content/_data/changelogs/lts.yml

```diff
@@ -4389,6 +4389,331 @@
+    category: rfe
+    pull: 4920
+    issue: 60972
+    authors:
+    - romenrg
+    message: |-
+      Name the first breadcrumb "Dashboard" for clarity.
+
+  - type: rfe
+    category: rfe
+    pull: 5016
+    authors:
+    - daniel-beck
+    message: |-
+      Remove TFS Plugin suggestion from Setup Wizard.
+
+  - type: rfe
+    category: rfe
+    pull: 4952
+    authors:
+    - daniel-beck
+    message: |-
+      Disable autocomplete of username on login form.
+
+  - type: rfe
+    category: rfe
+    pull: 4950
+    issue: 63795
+    authors:
+    - aHenryJard
+    message: |-
+      Allow users with the <code>Jenkins.MANAGE</code> permission to restart and safe restart Jenkins.
+
+  - type: rfe
+    category: rfe
+    pull: 4905
+    authors:
+    - tszmytka
+    message: |-
+      Do not render parts of a serialized <code>ConsoleNote</code> in truncated log output.
+
+  - type: rfe
+    category: rfe
+    pull: 4910
+    authors:
+    - daniel-beck
+    references:
+      - pull: 4910
+      - url: "https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cross-Origin-Opener-Policy"
+        title: Cross-Origin-Opener-Policy at developer.mozilla.org
+    message: |-
+      Set <code>Cross-Origin-Opener-Policy</code> to <code>same-origin</code>.
+
+  - type: rfe
+    category: rfe
+    pull: 4911
+    authors:
+    - jglick
+    message: |-
+      Avoid losing work in progress by using a new browser tab to open the plugin link in a configuration screen’s inline help.
+
+  - type: rfe
+    category: rfe
+    pull: 4027
+    issue: 57484
+    authors:
+    - Wadeck
+    message: |-
+      Improve the scripting capacity related to the API Token system.
+      Provide a way to configure a fixed/default API Token for admin during installation phase.
+
+  - type: rfe
+    category: localization
+    pull: 4995
+    authors:
+    - SoleneGK
+    references:
+      - pull: 4995
+      - pull: 4996
+      - pull: 4997
+    message: |-
+      Add missing French localization for the login page, setup wizard and the main page.
+
+  - type: rfe
+    category: rfe
+    pull: 4904
+    authors:
+    - takashiharano
+    message: |-
+      Add Japanese translations for user configuration screen.
+
+  - type: bug
+    category: bug
+    pull: 4978
+    authors:
+    - PhilippHomann
+    references:
+      - pull: 4978
+      - url: "https://www.jenkins.io/doc/book/managing/system-properties/#historywidget-descriptionlimit"
+        title: Features controlled with System Properties
+    message: |-
+      Hide description panel in sidebar if historyWidget.descriptionLimit is 0.
+
+  - type: bug
+    category: bug
+    pull: 4889
+    issue: 42228
+    authors:
+    - takashiharano
+    message: |-
+      Prevent JavaScript error when registering validators in some cases.
+
+  - type: bug
+    category: bug
+    pull: 4881
+    authors:
+    - daniel-beck
+    message: |-
+      Do not block rendering of Manage Jenkins while waiting for update center data.
```

Comment:
too minor

Reply:
Thanks.  Removed in a6adc4203a51928bc2e3f233f4d041baa078bd35